package.json

```diff
@@ -1025,7 +1067,8 @@
     "deploy:vsce": "vsce publish --packagePath ocaml-platform.vsix --yarn",
     "deploy:ovsx": "ovsx publish --yarn",
     "fmt:check": "prettier . --check",
-    "fmt": "prettier . --write"
+    "fmt": "prettier . --write",
+    "web": "npx @vscode/test-web --extensionDevelopmentPath=. ."
```

Comment:
Should this be named `test:web` if it's running tests?

Reply:
It's not running tests, just launching vscode in a web browser with the web extension. But yes I used this to test the extension so it could be prefixed by "test:"

Reply:
Thinking about it, it is not a good fit because `"test": "npm-run-all -s test:*",` would then run it even if it's not an automated test target. 

I have added a commit to change the name to `run:web` but feel free to suggest something else.